cfgov/unprocessed/apps/tccp/js/index.js

```diff
@@ -134,7 +138,11 @@ function handleShowMore(event) {
   }
   const results = document.querySelector('.o-filterable-list-results');
   const showMoreFade = document.querySelector('#u-show-more-fade');
-  const nextResult = document.querySelector('.u-show-more > a');
+  const nextResult = document.querySelector('[data-js-hook="behavior_faded-card"]');
+  nextResult.setAttribute('tabIndex', '0');
+  nextResult.querySelectorAll('[tabindex="-1"]').forEach( elem => {
+    elem.setAttribute('tabIndex', '0');
+  });
```

Comment:
I think this can just be 
```suggestion
  nextResult.querySelector('[tabindex="-1"]').setAttribute('tabIndex', '0');
```